extensions/vertx-keycloak/runtime/src/main/java/io/quarkus/vertx/keycloak/KeycloakConfig.java

```diff
@@ -0,0 +1,313 @@
+package io.quarkus.vertx.keycloak;
+
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class KeycloakConfig {
```

Comment:
Wondering if we should make it `VertxKeycloak`. To be honest, I'm not sure how much we want people to be aware of Vert.x.

But I suppose all this go away for RESTEasy as soon as you include the Undertow dependency?

Reply:
Hi @gsmet I was hoping to convince Stuart to keep 'vertx-' away from the module name :-), but may be it is not a big deal :-)